site/documentation/content/api/Metrics.md

```diff
@@ -75,6 +75,7 @@ Metrics provided by the protocol adapters are:
 | *hono.connections.authenticated*   | Gauge               | *host*, *component-type*, *component-name*, *tenant*                                         | Current number of connected, authenticated devices. <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
 | *hono.connections.unauthenticated* | Gauge               | *host*, *component-type*, *component-name*                                                   | Current number of connected, unauthenticated devices. <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
 | *hono.connections.authenticated.duration* | Timer        | *host*, *component-type*, *component-name*, *tenant*                                         | The overall amount of time that authenticated devices have been connected to protocol adapters. <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
+| *hono.connections.rejected.total*  | Counter             | *host*, *component-type*, *component-name*, *tenant*                                         | The number of attempted connections to protocol adapters which were rejected because a configured connection limit was hit. <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
```

Comment:
I would rather replace this with a metric for connection attempts in general. We could then use tags to indicate the outcome like *succeeded*, *unauthenticated*, *rejected* (due to resource limitations). In any case, this metric would only make sense as a connection attempt rate, i.e. how many attempts per time-period instead of absolute numbers.

Reply:
Since the number of rejected connections is a number which can only increase but never decrease I deemed a Counter to be the appropriate type as the alternatives listed by https://prometheus.io/docs/concepts/metric_types/ didn't really fit.

Isn't the `range` function of Prometheus (https://prometheus.io/docs/prometheus/latest/querying/functions/#rate) which converts this to a rate, providing the metric as you want it to be?

Reply:
I forgot: i think having this as a more generic metric with a label is a good idea! Is this something you see as part of this issue or should I create a follow-up issue?

Reply:
I reworked the metric using labels, see commit 3d3ce96.

This doesn't cover all outcomes like you mentioned above, but it's a start and we can continue to add more labels as needed. 

@ctron Would this also cover your comment?